toolkit/tools/go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/microsoft/azurelinux/toolkit/tools
 
-go 1.21
+go 1.23.0
 
-toolchain go1.21.6
+toolchain go1.24.2
```

Comment:
1.24.2 or 1.23.?

Reply:
This is auto generated by the go tools, it matches the exact version of the tools used at that time.